src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,12 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+
+export const TodoList = ({ todos }) => (
+  <>
+    {todos.map(todo => (
+      <section className="TodoList">
+        <TodoInfo todo={todo} key={todo.id} />
+      </section>
```

Comment:
<img width="531" alt="image" src="https://github.com/mate-academy/react_static-list-of-todos-js/assets/70220667/f365f731-eb08-4274-8f4c-7e5236fa83d8">
i think you should have only one todoList section